buildone

```diff
@@ -41,54 +41,66 @@ log() {
     echo "$@" >&2
 }
 
+is_snapshot() {
+    local -r dist_snapshot_regex="^(jessie|stretch|buster|unstable)-snapshot"
+    [[ $1 =~ $dist_snapshot_regex ]]
+}
+
 build() {
     DIST=$1
+    if is_snapshot "$DIST"; then
+        ! debian_snapshot_id=$(cat build/snapshot_id) && return
+        TAG="${DIST}-${debian_snapshot_id}"
+    else
+        TAG=$DIST
+    fi
+
```

Comment:
Rather than doing this I suggest taking another parameter that is the snapshot id, like the `mkimage` script.

This should avoid needing the `debootstrap/buster-snapshot` file.

Reply:
we would like to have both images released so if we remove this buster-snapshot file we will only have 1 since we are iterating over 

```
DISTS="jessie
stretch
buster
buster-snapshot
unstable
"
```

if we use just `buster` with a parameter we lose this functionality where we can choose when we want to have a snapshot or what distro would have a snapshot because we would have to do something like:

```
for DIST in $DISTS; do
    ./buildone "$DIST"
    ./buildone "$DIST" "$snapshot_id"
done

```

Reply:
yeah, sorry, I would also suggest adding something like
```
SNAPSHOT_DISTS="buster"

for DIST in $SNAPSHOT_DISTS; do
    ./buildone "$DIST" "$snapshot_id"
done
```

so that you build the same images as before, plus the snapshot images.

Reply:
Ok, now I see you point! I will try sth... ;)

Reply:
done!